### ISSUE TITLE
Can't create client on python 3.4.8

### ISSUE THREAD

**Issue by dmitriyshashkin** [dask/distributed#2045]
I've installed the latest versions of dask and distributed using pip. I'm trying to run the simples code sample:
```
from dask.distributed import Client
client = Client()
```
and I get the following error:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.4/site-packages/distributed/client.py", line 578, in __init__
    self.start(timeout=timeout)
  File "/usr/lib/python3.4/site-packages/distributed/client.py", line 698, in start
    sync(self.loop, self._start, **kwargs)
  File "/usr/lib/python3.4/site-packages/distributed/utils.py", line 253, in sync
    six.reraise(*error[0])
  File "/usr/lib/python3.4/site-packages/six.py", line 693, in reraise
    raise value
  File "/usr/lib/python3.4/site-packages/distributed/utils.py", line 238, in f
    result[0] = yield make_coro()
  File "/usr/lib/python3.4/site-packages/distributed/utils.py", line 220, in make_coro
    coro = gen.maybe_future(func(*args, **kwargs))
  File "/usr/lib64/python3.4/site-packages/tornado/gen.py", line 335, in wrapper
    runner = Runner(result, future, yielded)
  File "/usr/lib64/python3.4/site-packages/tornado/gen.py", line 1044, in __init__
    if self.handle_yield(first_yielded):
  File "/usr/lib64/python3.4/site-packages/tornado/gen.py", line 1187, in handle_yield
    self.future = convert_yielded(yielded)
  File "/usr/lib64/python3.4/functools.py", line 727, in wrapper
    return dispatch(args[0].__class__)(*args, **kw)
  File "/usr/lib64/python3.4/site-packages/tornado/gen.py", line 1327, in convert_yielded
    return _wrap_awaitable(yielded)
  File "/usr/lib64/python3.4/asyncio/tasks.py", line 553, in ensure_future
    raise TypeError('A Future, a coroutine or an awaitable is required')
TypeError: A Future, a coroutine or an awaitable is required
```
Perhaps it has something to do with my python version, as I have no problem when I install dask with conda together with python 3.6

**Comment by mrocklin** [comment]
For what it's worth the distibuted scheduler doesn't explicitly provide
support or test against Python 3.4, which we've found is rarely used.

**Comment by dmitriyshashkin** [comment]
@mrocklin thank you for your prompt response. I'll try to update python to more up to date version. I'd suggest adding python version to requirements though

**Comment by mrocklin** [comment]
It looks like we only list 2.7, 3.5, and 3.6 on PyPI:
https://pypi.org/project/distributed/

**Comment by mrocklin** [comment]
Closing this for now.  Please let me know if it should be reopened.

**Comment by TomAugspurger** [comment]
More recent versions of setuptools have a `python_requires`, though [the syntax is a bit awkward](https://github.com/pandas-dev/pandas/blob/480790531ffcc4329f280ddf6877d028d08e969f/setup.py)

**Comment by TomAugspurger** [comment]
This will become important when distributed goes python3 only, so that 2.7 people get the latest version that included 2.7 in the `python_requires`.
